src/lib/components/Editor.svelte

```diff
@@ -72,19 +74,26 @@
 
 	$: extensions = get_extensions($editor_config);
 
-	function read_current_tab(current_tab: string) {
+	function read_current_tab(current_tab: string, is_image: boolean) {
 		if (!current_tab) return;
-		webcontainer.read_file(current_tab).then((file) => {
-			code = file;
+		if (!is_image) {
+			webcontainer.read_file(current_tab).then((file) => {
+				code = file;
+			});
+			return;
+		}
+		webcontainer.read_file(current_tab, false).then((file) => {
+			image_bytes = file;
 		});
 	}
 
 	$: current_lang = $current_tab.split('.').at(-1) ?? 'svelte';
+	$: is_image = ['png', 'bmp', 'jpg', 'jpeg', 'gif'].includes(current_lang);
```

Comment:
I'd love support for .svg and .webp as well 👀

Reply:
svg are text based so the editor needs to be in text form and not in image form...i'll add webp tho